src/Routes.js

```diff
@@ -57,6 +58,7 @@ const Routes = (props) => (
     <Route path="/home_researcher_info" component={HomeResearcherInfo} />
     <Route path="/election404" component={Election404} />
     <Route path="/NHGRIpilotinfo" component={NHGRIpilotinfo} />
+    <Route path="/NIHICWebform" component={NIHICWebform} />
```

Comment:
It's preferable to keep urls all lower case. It's less of a concern in our world as we have everything deployed to linux based hosts, but casing mistakes are easy to make especially in one-word-paths.
```suggestion
    <Route path="/nih_ic_webform" component={NIHICWebform} />
```

Reply:
Can you make this change?